### DISCUSSION TITLE
is there ability to write unit tests?

### DISCUSSION THREAD

**Discussion by artemklishch** [nuxt/nuxt#23395] [Questions]
Hi!
i saw in documentation that testing in browser still is in the development. But particularly unit tests can be written.
Can I write unit test to check the call of spy function for the moment? I mean, the call of Pinia action, or function from the different file.
Is there examples anywhere? I tried to follow the Vitest documentation and it didn't work.
Thanks.

**Comment by TheAlexLichter** [comment]
Have a look at https://github.com/danielroe/nuxt-vitest

**Reply by artemklishch** [reply]
I tried to use 'nuxt-vitest'. Unfortunately, it didn't work.
And now, on the default project, created with 'npx nuxi@latest init <name>' and next following instructions on the 'nuxt-vitest' npm page (they are almost the same as in the above link) - I get the error - ' Vitest  "deps.inline" is deprecated. If you rely on vite-node directly, use "server.deps.inline" instead. Otherwise, consider using "deps.optimizer.ssr.include"'
